src/main/java/com/softserve/academy/spaced/repetition/logger/LogException.java

```diff
@@ -1,27 +1,27 @@
 package com.softserve.academy.spaced.repetition.logger;
 
+import com.softserve.academy.spaced.repetition.Application;
 import org.aspectj.lang.JoinPoint;
 import org.aspectj.lang.annotation.AfterThrowing;
 import org.aspectj.lang.annotation.Aspect;
-import org.springframework.beans.factory.annotation.Autowired;
+import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Component;
+import org.slf4j.Logger;
 
 @Aspect
 @Component
 public class LogException {
-    @Autowired
-    Logger logger;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(LogException.class);
 
     @AfterThrowing(value = "execution(* com.softserve.academy.spaced.repetition.controller.*.*(..))", throwing = "ex")
     public void loggingExceptionsFromControllers(JoinPoint joinPoint, Exception ex) {
-        System.out.println(logger);
-        logger.log("Exception in controller = " + joinPoint.getThis() + ", method = " + joinPoint.getSignature());
+        LOGGER.trace("Exception in controller = " + joinPoint.getThis() + ", method = " + joinPoint.getSignature());
```

Comment:
One more thing. 

- Exceptions should be logged at ERROR level.
- Sonar will report somethig like "...conditional formatting should be used..."
- the actual exception should be logged.

Please rewrite as 
`LOGGER.error("Exception in controller = {} method = {}", joinPoint.getThis(), joinPoint.getSignature(), ex);`

The same below.